examples/basic.pp

```diff
@@ -1,6 +1,17 @@
 require epel
 
-Yumrepo['epel'] -> Class['htcondor']
+Yumrepo['epel'] -> Class['yum'] -> Class['htcondor']
```

Comment:
The `yum` class doesn't depend on the `epel` yumrepo resource and doesn't need to be forced to come before it in the catalog.

```suggestion
Yumrepo['epel'] -> Class['htcondor']
Class['yum'] -> Class['htcondor']
```

Reply:
Totally right, didn't knew how to split the dependency 👏🏻